apps/laboratory/tests/connect.spec.ts

```diff
@@ -31,3 +31,15 @@ testMW(
     await modalValidator.expectRejectedSign()
   }
 )
+
+testMW(
+  'it should switch networks and sign',
+  async ({ modalPage, walletPage, modalValidator, walletValidator }) => {
+    const targetChain = 'Polygon'
+    await modalPage.switchNetwork(targetChain)
```

Comment:
This test doesn't ensure that the network wasn't already on Polygon. It might be good to switch between multiple networks and expect multiple signatures, e.g. switch to Polygon, sign, expect Polygon signature, then switch to ETH Mainnet, sign, expect ETH Mainnet signature